frontends/web/src/components/forms/input.module.css

```diff
@@ -49,7 +49,7 @@
 }
 
 .input input[disabled] {
-    background-color: var(--color-white) !important;
+    background-color: var(--background-secondary) !important;
     border-color: var(--color-lightgray) !important;
```

Comment:
change `--color-lightgray` to `--background` ?

Reply:
It's a `border-color`, so I wasn't sure whether borders should use `font colors` or `background colors`. So I just left it how it is. But I think `border-color` can use background colors if we decide that's ok.